docs/mkdocs.yml

```
@@ -7,7 +7,26 @@ nav:
   - At Sea Operation: atseaoperation.md
   - ROS 2 Interface: ros2.md
   - Simulation: simulation.md
-  - Tutorials: tutorials.md
+  - Tutorials:
+      - Install:
+          - Install (from source): Tutorials/Install/Install_source.md
+          - Install (Docker): Tutorials/Install/Install_docker.md
+      - Simulation:
+          - Run the Simulator: Tutorials/Simulation/RunSimulator.md
+          - View ROS 2 Messages: Tutorials/Simulation/SimulatorOutputROS.md
+          - View ROS 2 Messages with Plotjuggler: Tutorials/Simulation/SimulatorOutputPlotjuggler.md
+          - Simulator Output Data Logs: Tutorials/Simulation/SimulatorOutputLogs.md
+          - Control Simulator with pbcmd: Tutorials/Simulation/SimulatorInteractionPbcmd.md
+          - Adjust Simulator parameters: Tutorials/Simulation/SimulatorParameters.md
+      - ROS2:
```

I think it would be good to keep the original heading, as it is more specific.
```suggestion
      - Adding Control Code:
```